paint-timing/resources/utils.js

```diff
@@ -65,3 +65,23 @@ function assertFirstContentfulPaint(t) {
     });
   });
 }
+
+function test_fcp(label) {
+  promise_test(async t => {
+    try {
+      assert_precondition(window.PerformancePaintTiming, "Paint Timing isn't supported.");
+      const main = document.getElementById('main');
+      await new Promise(r => window.addEventListener('load', r))
+      await assertNoFirstContentfulPaint(t)
+      main.className = 'intermediate'
+      await assertNoFirstContentfulPaint(t)
+      main.className = 'contentful'
+      await assertFirstContentfulPaint(t)
+    } catch (e) {
+      document.getElementById('log').style.display = 'block'
```

Comment:
@foolip is this something we should expect to do in any rendering tests to avoid logs from changing the results of the tests? Is this the recommended way of solving this issue (the issue here being that text logs from the text will trigger FCP)? Is there any plan to surface the text logs only until after the test is done?